action/protocol/context.go

```diff
@@ -76,6 +79,16 @@ type (
 		Nonce uint64
 	}
 
+	// TestCtx is ctx used in test or tool
+	TestCtx struct {
+		// DisableCheckIndexer disables to check indexer
+		DisableCheckIndexer bool
```

Comment:
don't need this flag?
it's always true for both the `play` and `commit` command

Reply:
in other words, the existence of `TestCtx` indicates user wants to disable check indexer

Reply:
fixed